core/src/View/Sound.java

```diff
@@ -0,0 +1,55 @@
+package View;
+
+import Model.Enemies.Mouse;
+import Model.Entity;
+import Model.Model;
+import com.badlogic.gdx.Gdx;
+
+import java.util.ArrayList;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+
+public class Sound{
+    public com.badlogic.gdx.audio.Sound getSound() { //TODO generalise for everything
+
+        com.badlogic.gdx.audio.Sound sound = Gdx.audio.newSound(Gdx.files.internal("Enemies/Mouse/mouse-squeek.mp3"));
+        return sound;
+    }
+
+    public void playSounds(final Model model) {
```

Comment:
Documentation for this method could be added.

Reply:
Yes sir